<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientDialog</class>
 <widget class="QDialog" name="GradientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gradient Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="GradientEdit" name="gradient_edit" native="true">
     <property name="minimumSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="type_group">
     <property name="title">
      <string>Gradient Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="type_linear_rbtn">
        <property name="text">
         <string>Linear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type_radial_rbtn">
        <property name="text">
         <string>Radial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type_conical_rbtn">
        <property name="text">
         <string>Conical </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="spread_group">
     <property name="title">
      <string>Spread Method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="spread_pad_rbtn">
        <property name="text">
         <string>Pad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="spread_repeat_rbtn">
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="spread_reflect_rbtn">
        <property name="text">
         <string>Reflect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="GradientStopsEdit" name="stops_edit" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GradientEdit</class>
   <extends>QWidget</extends>
   <header>gradient_edit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GradientStopsEdit</class>
   <extends>QWidget</extends>
   <header>gradient_stops_edit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GradientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GradientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>type_conical_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>spread_group</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
